docker/app/Dockerfile

```diff
@@ -0,0 +1,15 @@
+from node:lts-alpine
+
+WORKDIR /home/www
+
+ADD package.json ./package.json
+RUN npm install
+
+ADD public ./public
```

Comment:
Any reason we can't do `ADD app .` or something like that? I don't mind it being individual, but I want to make sure I can do the most minimal maintenance in docker, and be able to make changes to the code structure without any unexpected side-effects (like docker stops working).

Reply:
I chose not to go this route as the image could accidentally include something that was unintended. I've seen secrets or keys were accidentally included before when someone published to a docker repository, so my default behavior is to be minimal.

If you'd prefer to copy everything it's an easy change. I'll include a .dockerignore as well to ensure the process doesn't attempt to copy things like local node_modules